packages/create-react-native-library/templates/native-common/android/build.gradle

```diff
@@ -25,6 +25,9 @@ def isNewArchitectureEnabled() {
 
 apply plugin: "com.android.library"
 apply plugin: "kotlin-android"
+<% if (project.nitro) { -%>
+apply from: '../nitrogen/generated/android/<%- project.package_cpp -%>+autolinking.gradle'
```

Comment:
Is this file generated by `nitrogen`? Then it also needs to be in the build script so that it's run every time before publishing the package.

Also shouldn't this be in `.gitignore`?